docs/docs-next/docs/guides/automation.md

```diff
@@ -170,8 +105,8 @@ Use this table to help guide your decision:
 
 ## Next Steps
 
-- Learn more about [advanced scheduling patterns](link-to-advanced-scheduling)
-- Explore [complex sensor examples](link-to-sensor-examples)
-- Dive into [Declarative Automation best practices](link-to-declarative-automation)
+- Learn more about [advanced scheduling patterns]
+- Explore [complex sensor examples]
```

Comment:
Similar comment here - what does `complex` mean? I don't want us to include code in here, but without understanding what a simple sensor looks like, I don't know what to expect for something being labeled as complex.

Why not just `Explore some sensor examples` or something similar?